### ISSUE TITLE
Default font-size should be much higher

### ISSUE THREAD

**Issue by AntoninGrele** [sirodoht/mataroa#1]
Hello ! I like your project, but I have a recommendation. Currently it looks like the font size is 14px.
It's also set twice, both in `html `and in `body`. `html` should be enough.

We write blogs for them to be read ! For desktop screens, 21px is much more readable than 14px. For smaller screens, we tend to be closer to them when we read, so 16px would be ideal.

**Comment by sirodoht** [comment]
Hi Antonin, thank you for your comment! I'm actually still thinking about this. I feel the small letters give off a certain vibe that I like. I don't find it less readable, but if many people do, I'll reconsider it.

**Comment by AntoninGrele** [comment]
It's very common for graphic designers to make designs that look nice, but have terrible usability or readability, so your point is a common one !

Think about it this way : if I'm reading your blog while my laptop is on a table and I'm sitting in the back of my chair, which makes my eyes about 1 meter away from the screen, is it _comfortable_ to read ?

Even with good eyesight, 14px is not comfortable for laptop/desktop sizes. It's ok for minor text, but not full articles. 16px is the default browser size, and I think that's an OK minimum, but 18px / 21px is better, imho.

Medium uses a base size of 21px, for example.

You can set your column size in rem to make it scale with your font size based on the screen size, so that you keep the same look as what you had in mind :)

**Comment by asvc** [comment]
On the contrary, I find the default font size just right. How about a custom CSS support [https://github.com/sirodoht/mataroa/issues/3] instead of trying to manufacture a "one size fits all" solution?

**Comment by alecstein** [comment]
I strongly agree with either increasing the font size or adding custom .css. I have good vision and I almost never read font this small. I cannot imagine anyone I know wanting to read my blog if they have to squint ;-) 

But I'm so happy with mataroa for other reasons, I can't really complain. Great work my dude!

**Comment by sirodoht** [comment]
I have just added a second theme with a larger font / layout. Still tweaking it, but v1 is live.

To enable go to Dashboard -> Blog settings -> Theme Zia Lucia

**Comment by sirodoht** [comment]
No further complains about font size have occurred in the last five years after the change.
